packages/request-tracker/lib/request-tracker-xhr.ts

```diff
@@ -14,21 +14,21 @@ interface RequestData {
 
 type ReadyStateChangeHandler = (this: XMLHttpRequest, ev: Event) => any
 
-function createXmlHttpRequestTracker (window: WindowWithXmlHttpRequest, clock: Clock): RequestTracker {
+function createXmlHttpRequestTracker (global: GlobalWithXmlHttpRequest, clock: Clock): RequestTracker {
```

Comment:
This should probably take `XMLHttpRequest` and `document?` separately — the fetch version needs a global object because it overwrites `global.fetch` directly, but this version can work from only the `XMLHttpRequest` object as it overwrites the prototype

It's pretty confusing for `react-native/lib/client.ts` to pass a new object with XHR — it looks like that should never work because overwriting XHR on that object would do nothing